src/utils/formatPeriod.js

```diff
@@ -0,0 +1,45 @@
+import { differenceInDays, format } from 'date-fns';
+import nl from 'date-fns/locale/nl-BE';
+import fr from 'date-fns/locale/fr';
+import { capitalize } from 'lodash';
+
+const locales = {
+  nl,
+  fr,
+};
+
+const formatPeriod = (startDate, endDate, locale, t) => {
+  const start = new Date(startDate);
+  const end = new Date(endDate);
+
+  const formatTypeDate = 'EEEE d MMMM yyyy';
+  const formatTypeTime = 'HH:mm';
+
+  const formattedDateStart = format(start, formatTypeDate, {
+    locale: locales[locale],
```

Comment:
Why don't we just pass the `locale` here? `locales[nl]` always evaluates to `nl`, no? And same for `fr`?

Reply:
The locales used here are the translations from the `date-fns` library.

```
import nl from 'date-fns/locale/nl-BE';
import fr from 'date-fns/locale/fr';
```